test/javascripts/acceptance/question-answer-test.js

```diff
@@ -299,6 +304,35 @@ acceptance("Discourse Question Answer - logged in user", function (needs) {
     );
   });
 
+  test("validations for comment length", async function (assert) {
+    await visit("/t/280");
+    await click("#post_1 .qa-comment-add-link");
+
+    await fillIn(".qa-comment-composer-textarea", "a".repeat(4));
+
+    assert.strictEqual(
+      query(".qa-comment-composer-flash").textContent.trim(),
+      I18n.t("qa.post.qa_comment.composer.too_short", { count: 5 }),
+      "displays the right message about raw length when it is too short"
+    );
+
+    await fillIn(".qa-comment-composer-textarea", "a".repeat(6));
+
+    assert.strictEqual(
+      query(".qa-comment-composer-flash").textContent.trim(),
+      I18n.t("qa.post.qa_comment.composer.length_ok", { count: 14 }),
+      "displays the right message about raw length when it is OK"
```

Comment:
Same comment as further up, just not sure whether we need to bother showing a "this is fine" message?

Reply:
O it isn't showing a `this is fine` message but more like 

```
            length_ok:
              one: "%{count} character left..."
              other: "%{count} characters left..."
```

Reply:
Cool thanks for clarifying :+1: